### PR TITLE
Fix question history

### PR DESCRIPTION
**Pull request by yongxiangng**
We now show matches, even if there are no code written/chat used

### CODE REVIEW

File: frontend/src/components/chat/ChatBody.js [hunk 1]
```diff
@@ -20,11 +20,11 @@ const ChatBody = ({ chatSocket, username, room_id, is_live }) => {
   })
 
   chatSocket.on('message response', (data) => {
-    console.log(data)
-    setMessages([...messages, data])
+    let newMessages = [...messages, data];
     if (is_live) {
-      axiosApiInstance.post(URL_USER_SVC_MESSAGE, {room_id: room_id, messages: messages})
+      axiosApiInstance.post(URL_USER_SVC_MESSAGE, {room_id: room_id, messages: newMessages})
```
Comment (ngzhenteng) [inline frontend/src/components/chat/ChatBody.js:25]: Nice, i happened to have written this in the branch "[origin/feat/room-page-layout](https://github.com/CS3219-AY2223S1/cs3219-project-ay2223s1-g27/tree/origin/feat/room-page-layout)" for the currently open PR as well.

File: frontend/src/components/chat/ChatBody.js [hunk 2]
```diff
@@ -20,11 +20,11 @@ const ChatBody = ({ chatSocket, username, room_id, is_live }) => {
   })
 
   chatSocket.on('message response', (data) => {
```
Comment (ngzhenteng) [inline frontend/src/components/chat/ChatBody.js:22]: I have actually moved the .on listeners to "RoomPage.js" to allow Chat to work with the "Popover" MUI component. No worries the code isn't too different.

Comment (yongxiangng) [reply frontend/src/components/chat/ChatBody.js:22]: Nw, we can wait for the other pr to be merged in then ill fix the differences

Comment (ngzhenteng) [reply frontend/src/components/chat/ChatBody.js:22]: Thank you a lot, sorry for inconveniences on your own

**Review by ngzhenteng — APPROVED** [review]
LGTM thank you.

**Pull request merged**